packages/siop-oid4vp/lib/authorization-request/AuthorizationRequest.ts

```diff
@@ -277,4 +289,8 @@ export class AuthorizationRequest {
   public async getPresentationDefinitions(version?: SupportedVersion): Promise<PresentationDefinitionWithLocation[] | undefined> {
     return await PresentationExchange.findValidPresentationDefinitions(await this.mergedPayloads(), version)
   }
+
+  public async getDcqlQuery(): Promise<DcqlQuery> {
+    return await findValidDcqlQuery(await this.mergedPayloads())
```

Comment:
I think `findValidDcqlQuery` can return `Promise<DcqlQuery | undefined>` so shouldn't this method also return `| undefined`?